docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -173,9 +173,9 @@
           {
             alert: 'NodeNetworkTransmitErrs',
             expr: |||
-              rate(node_network_transmit_errs_total[2m]) / rate(node_network_transmit_packets_total[2m]) > 0.01
+              rate(node_network_transmit_errs_total{%(nodeExporterSelector)s}[2m]) / rate(node_network_transmit_packets_total{%(nodeExporterSelector)s}[2m]) > 0.01
             ||| % $._config,
-            'for': '1h',
+            'for': '15m',
```

Comment:
I cannot really judge which time makes sense for network errors. @v-zhuravlev do you have a rationale for this? The corresponding commit doesn't have an explanation.

Reply:
Due to burstable nature of network traffic, I think it is very rare that alert would fire actually. moreover, as mentioned above by me, I think that 1h grace period is very long time and 15m should be enough to filter noise.